spec/requests/errors_spec.rb

```diff
@@ -60,4 +60,16 @@
       expect(response.body).to include(I18n.t!("errors.local_data_unavailable.title"))
     end
   end
+
+  it "forbids users without pre-release permission from accessing pre-release documents" do
```

Comment:
This test is good. I found it hard to review, though, because this request spec is inconsistent with all the others, which indicate the route they're testing using a `describe "METHOD /route"` block. 

I think would find this spec easier to understand if it were in a `describe` block that indicated the request being made. Since it could be anything, perhaps something like `describe "ANY /pre-release-document-path"`.